### DISCUSSION TITLE
21.6.8

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1426] [General]
## Releases Notes for 21.6.8
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/draw.io-21.6.8-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/draw.io-21.6.8-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/draw.io-universal-21.6.8.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/drawio-amd64-21.6.8.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/drawio-x86_64-21.6.8.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.8/drawio-x86_64-21.6.8.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 21.6.8](https://github.com/jgraph/drawio/blob/v21.6.8/ChangeLog). All changes from 21.6.6 to 21.6.8 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.6.8'>21.6.8</a>.</em>

**Comment by gongrui2233** [comment]
drawio中，画流程图中，在每个图标中加入一些数据，数据内容分别工作1，工作2，工作3，...工作10，工作11，最后填写完键值，编辑完成后，导出html网页时候，发现工作1，和工作10、工作11，在一起，然后再到工作2，工作3，特别无语
